geosnap/analyze/incs.py

```diff
@@ -108,8 +111,57 @@ def linc(labels_sequence):
         intersection = set.intersection(*neighbors)
         union = set.union(*neighbors)
         n_union = len(union)
-        if n_union == 1: # singleton at all points in time
-            lincs[i] = 0.
+        if n_union == 1:  # singleton at all points in time
+            lincs[i] = 0.0
         else:
-            lincs[i] = 1.0 - ((len(intersection)-1)/(n_union-1))
+            lincs[i] = 1.0 - ((len(intersection) - 1) / (n_union - 1))
     return lincs
+
+
+def lincs_from_gdf(
```

Comment:
@sjsrey could you take a look at this?

Reply:
(function name is a palceholder). I want to double check the interpretation is correct depending on the orientation...